src/frontend/src/expr/table_function.rs

```diff
@@ -79,13 +80,11 @@ impl TableFunction {
         let return_type = {
             // arguments:
             // file format e.g. parquet
-            // storage type e.g. s3
-            // s3 region
-            // s3 access key
-            // s3 secret key
-            // file location
-            if args.len() != 6 {
-                return Err(BindError("file_scan function only accepts 6 arguments: file_scan('parquet', 's3', s3 region, s3 access key, s3 secret key, file location)".to_owned()).into());
+            // storage type e.g. s3, gcs
+            // For s3: file_scan(file_format, s3, s3_region, s3_access_key, s3_secret_key, file_location_or_directory)
+            // For gcs: file_scan(file_format, gcs, credential, service_account, file_location_or_directory)
```

Comment:
Seems we can remove the `service_account`.

Reply:
My bad, forget to remove it.